README.md

```diff
@@ -0,0 +1,188 @@
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [JavaScript Capstone - TV Shows] <a name="about-project"></a>
+
+This project is about the creation of a dynamic app, that shows in a responsive way a different selection of actual TV Shows, and giving a selection on demand by applying a search with your input desired. 
+
+**[JavaScript Capstone - TV Shows]** For this project, we have put into practice all we have learned during the JavaScript module. It was also our first pair programing project to be run this way, following project management tools as Kanban board and daily updates. In the technical part, we have applied Js best practicies using ES6 syntax & modules, use webpack, Gitflow and use 2 different APIs.
+
+Find here video that shows the project details and functionality: (https://drive.google.com/file/d/174M16aVDuy5hUGUkCbcNxvBpU8lINgAJ/view?usp=drive_link)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Dynamic</summary>
+  <ul>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Standar code</summary>
+  <ul>
+    <li><a href="#">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Use GitFlow]**
+- **[Use Linters]**
+- **[Use ES6 syntax]**
+- **[Use JavaScript Modules]**
+- **[Use Webpack]**
+- **[Use Jest for testing]**
+- **[Use Async/await functions]**
+- **[Use TVmaze API]**
+- **[Use Microverse Involvement API]**
+- **[Use JSON]**
+- **[Use Kanban board to set project milestones and follow up]**
+- **[Work collaboratively doing pair programing]**
+- **[Add a user friendly & responsive design]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Open on Visual Studio Code or any other code reader, and use an app as Live Server to review online.
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/HFG43/JS_Capstone
+
+### Install
+
+Open index.html file with your Code Editor, and review the behaviour of the page under different sizes.
+
+### Usage
+
+To run the project, execute the following command:
+
+- In the command line:
```

Comment:
- [ ] [optional] It would have been nice if you added the command `npm install` that installs dependencies in package.json to make the app work.